tools/js-sdk-release-tools/src/test/packageVersion/changeClientFile.test.ts

```diff
@@ -0,0 +1,32 @@
+import * as fs from "fs";
+import * as path from "path";
+
+import { describe, expect, test } from 'vitest';
+import { join } from 'path';
+import { updateUserAgent } from '../../xlc/codeUpdate/updateUserAgent';
+
+describe('Update package version in /src', () => {
+    test('update package version for userAgentInfo with modular type',async ()=>{
+        const root = join(__dirname, 'testCases/modular-context/');
+        const expectedVersion = "1.0.0";
+        updateUserAgent(root,expectedVersion);
+        const data: string = fs.readFileSync(path.join(root, 'src',"api", "testContext.ts"), 'utf8');
+        expect(data.includes(expectedVersion)).toBe(true)
```

Comment:
please check the useragent line to be the expected one. e.g. `const userAgentInfo = \`azsdk-js-arm-edgezones/1.0.0-beta.4\`;`